CHANGES.rst

```diff
@@ -25,11 +25,12 @@ Changelog for zest.releaser
   Issue #55
   [maurits]
 
-- Add ``[ci skip]`` to commit messages to avoid running Travis
-  Continuous Integration builds.  See
-  http://docs.travis-ci.com/user/how-to-skip-a-build/
-  To activate this, add ``[zest.releaser] ci-skip = yes`` to the
-  ``setup.cfg`` of a package, or your global ``~/.pypirc``.
+- Optionally add extra text to commit messages.  This can be used to
+  avoid running Travis Continuous Integration builds.  See
+  http://docs.travis-ci.com/user/how-to-skip-a-build/.  To activate
+  this, add ``[zest.releaser] extra-message = [ci skip]`` to the
```

Comment:
Look like it belongs on one line. Perhaps re-phrase it as:

To activate this, add `extra-message = [ci skip]` to a `[zest.releaser]` section in your package's `setup.cfg`, or your....